src/core/utils/channel_monitor.hpp

```diff
@@ -168,6 +169,21 @@ class ChannelMonitor : public InstanceLocator
      */
     uint16_t GetChannelQuality(uint8_t aChannel) const;
 
+    /**
+     * This method finds the channel(s) with best quality (least interfered) in a given channel mask.
+     *
+     * The channels are compared based on their quality from `GetChannelQuality()` (i.e., percentage of RSSI samples
+     * higher than `kRssiThreshold`) and lower value (less interfered) is considered better.
+     *
+     * @param[in]  aMask         A channel mask (the search is limited to channels in @p aMask).
+     * @param[out] aQuality      A reference to `uint16` to return the quality value associated with best channel(s).
```

Comment:
Larger values is really lower quality, right?  Can we make the PR more intuitive and consistent?  For example, switch everything to "busy", "utilization", or "occupancy"?

Reply:
Will go for "occupancy". Probably will add a new commit (before the existing 3) to change this.